src/backend/slack/slack.ts

```diff
@@ -3,16 +3,17 @@ import fetch, { Response } from 'node-fetch';
 import { namespace } from '../config';
 import { logRequest, LOG_LEVEL } from '@navikt/familie-backend';
 import HttpsProxyAgent from 'https-proxy-agent/dist/agent';
+import { Agent } from 'node:http';
 
 const token = process.env.SLACK_TOKEN;
 const agent =
     process.env.ENV !== 'local' && process.env.ENV !== 'e2e'
-        ? new HttpsProxyAgent({
+        ? ((new HttpsProxyAgent({
               host: 'webproxy.nais',
               secureProxy: true,
               port: 8088,
               rejectUnauthorized: false,
-          })
+          }) as unknown) as Agent)
```

Comment:
På meget tynn is her, så tar gjerne inspill. Får typefeil uten casting, ref https://github.com/TooTallNate/node-https-proxy-agent/issues/98

Reply:
Jeg er usikker selv her om det er riktig casting og type, men jeg tenker at hvis det fjerner advarselen og ikke lager nye errors så er vi kanskje good 🤷🏻  
Kanskje @alexandraback  har noen innspill her? 🙈

Reply:
hva er forskjellen mellom HttpsProxyAgent og Agent ?

Reply:
```
(new Agent({
              host: 'webproxy.nais',
              secureProxy: true,
              port: 8088,
              rejectUnauthorized: false,
          })
```

Er dette gyldigt  liksom?

Reply:
Ser ut som `Any` er den offisielle løsningen på problemet :)